apiserver/objects_legacy_test.go

```diff
@@ -235,7 +233,7 @@ func (s *putCharmObjectSuite) TestUploadBumpsRevision(c *gc.C) {
 }
 
 func (s *putCharmObjectSuite) TestUploadVersion(c *gc.C) {
-	expectedVersion := "testcharm-146-g725cfd3-dirty"
+	expectedVersion := "dummy-146-g725cfd3-dirty"
```

Comment:
The test charm's name here is actually `dummy`, not `testcharm`, as you can see 3 lines below